src/functions/truthy.ts

```diff
@@ -4,7 +4,7 @@ export const truthy: IFunction = (targetVal, _opts, paths): void | IFunctionResu
   if (!targetVal) {
     return [
       {
-        message: `${paths.target ? paths.target.join('.') : 'property'} is not truthy`,
+        message: '{{fullPath}} property is not truthy',
```

Comment:
In what situation would one use `path` instead of `fullPath` when creating rules? We should be careful about introducing new properties here - we should include clear use case examples for each property in the docs - if we can't do that then it might be indicator that we can remove / combine properties.

Reply:
This is a part of a broader issue I'm addressing at the moment.
I'll revert that part of code code, just wanted to see how many code changes would be needed and to gather some feedback from you all.

https://github.com/stoplightio/spectral/commit/0ef818a843791a6fe9f544a8244484a961256b6d will address https://github.com/stoplightio/spectral/issues/815, but since there will be a couple of tests that I'll need to update, I was planning to do it separately.

Reply:
Reverted. To be addressed as a part of #815